packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx

```diff
@@ -106,6 +108,14 @@ function Sidebar(): JSX.Element {
 
         const [tree] = initTree([navigationRoot], results);
 
+        const mxPlanNode = findNodeById(tree, 'mxplan');
+        if (
+          mxPlanNode &&
+          environmentPlugin.getEnvironment().getRegion() === 'CA'
```

Comment:
Why not use feature flipping here ?

Reply:
email:mxplan feature is available for both EU & CA region (it's already feature flipped) but in EU the manager route is #/email_domai, in CA it's #/email_mxplan